symforce/examples/bundle_adjustment_in_the_large/bundle_adjustment_in_the_large.cc

```diff
@@ -0,0 +1,112 @@
+/* ----------------------------------------------------------------------------
+ * SymForce - Copyright 2022, Skydio, Inc.
+ * This source code is under the Apache 2.0 license found in the LICENSE file.
+ * ---------------------------------------------------------------------------- */
+
+#include <fstream>
+
+#include <sym/pose3.h>
+#include <symforce/opt/factor.h>
+#include <symforce/opt/internal/logging_configure.h>
+#include <symforce/opt/optimizer.h>
+#include <symforce/opt/values.h>
+
+#include "./gen/keys.h"
+#include "./gen/snavely_reprojection_factor.h"
+
+using namespace sym::Keys;
+
+sym::Factord MakeFactor(int camera, int point, int pixel) {
+  return sym::Factord::Hessian(sym::SnavelyReprojectionFactor<double>,
+                               {
+                                   sym::Key::WithSuper(CAM_T_WORLD, camera),
+                                   sym::Key::WithSuper(INTRINSICS, camera),
+                                   sym::Key::WithSuper(POINT, point),
+                                   sym::Key::WithSuper(PIXEL, pixel),
+                                   EPSILON,
+                               },
+                               {
```

Comment:
maybe we add a prefix like `/* optimized_keys */` for clarity

Reply:
Done